scripting/shavit-misc.sp

```diff
@@ -2085,6 +2092,92 @@ public Action Shotgun_Shot(const char[] te_name, const int[] Players, int numCli
 	return Plugin_Stop;
 }
 
+public Action EffectDispatch(const char[] te_name, const Players[], int numClients, float delay)
+{
+	int iEffectIndex = TE_ReadNum("m_iEffectName");
+	int nHitBox = TE_ReadNum("m_nHitBox");
+	char sEffectName[64];
```

Comment:
Could you make this initialize like other `char[]`s?

```sp
char[] sEffectName = new char[32];
```

I also prefer 32 because I don't think there's any longer effect name. - do the same for the Get...Names!